stepper-lib/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "stepper-lib"
-version = "0.2.2"
+version = "0.2.3"
```

Comment:
I think this deserves a minor version increment to express major change in behavior

Reply:
Sounds fair